apps/client/src/features/control/playback/tap-button/TapButton.tsx

```diff
@@ -13,14 +13,15 @@ interface TapButtonProps {
   theme?: Playback | 'neutral';
   active?: boolean;
   className?: string;
+  name?: string;
 }
 
 const TapButton = forwardRef((props: PropsWithChildren<TapButtonProps>, ref: ForwardedRef<HTMLButtonElement>) => {
-  const { children, disabled, onClick, theme = 'neutral', aspect = 'normal', active, className } = props;
+  const { children, disabled, onClick, theme = 'neutral', aspect = 'normal', active, className, name } = props;
   const classes = cx([style.tapButton, className, style[theme], style[aspect], active ? style.active : null]);
 
   return (
-    <button className={classes} disabled={disabled} type='button' onClick={onClick} ref={ref}>
+    <button className={classes} disabled={disabled} type='button' onClick={onClick} ref={ref} name={name}>
       {children}
     </button>
   );
```

Comment:
suggestion: what if we modify the type signature to extend button? that would make it easier to consume and compose. 

like 
```ts

interface TapButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
  aspect?: 'normal' | 'square' | 'fill' | 'tight';
  free?: boolean;
  theme?: Playback | 'neutral';
  active?: boolean;
}
```

then we consume like
```ts
const TapButton = () => {
  const { children, theme = 'neutral', aspect = 'normal', active, className, ...elementProps } = props;
  ...
  return (
    <button {...elementProps} className={classes} type='button' ref={ref}>
      {children}
    </button>
  );
}
```

This code also makes it clearer for us to understand what are the properties we override